test/fields/collections/Select/index.ts

```diff
@@ -221,6 +223,26 @@ const SelectFields: CollectionConfig = {
         },
       ],
     },
+    {
+      name: 'selectWithJsxLabelOption',
+      label: 'Select with JSX label option',
+      type: 'select',
+      defaultValue: 'three',
+      options: [
+        {
+          label: 'Value One',
+          value: 'one',
+        },
+        {
+          label: 'Value Two',
+          value: 'two',
+        },
+        {
+          label: CustomJSXLabel,
```

Comment:
How does this work when all other custom components use strings and go through the importMap?